client/dashboard/data/site.ts

```diff
@@ -88,7 +88,7 @@ export interface Site {
 	};
 	plan?: SitePlan;
 	capabilities: SiteCapabilities;
-	subscribers_count: number;
+	subscribers_count?: number; // Can be undefined query cache is prefilled from old Calypso Redux store.
```

Comment:
I think it's missing an `if`


```suggestion
	subscribers_count?: number; // Can be undefined if query cache is prefilled from old Calypso Redux store.
```